tests/integ/scala/test_function_suite.py

```diff
@@ -1709,6 +1710,31 @@ def test_array_to_string(session):
     )
 
 
+@pytest.mark.parametrize("col_amount", ["amount", col("amount")])
+def test_array_unique_agg(session, col_amount):
+    assert sorted(
+        json.loads(
+            TestData.monthly_sales(session)
+            .select(array_unique_agg(col_amount))
+            .collect()[0][0]
+        )
+    ) == [
+        200,
+        400,
+        800,
+        2500,
+        3000,
+        4500,
+        5000,
+        6000,
+        8000,
+        9500,
+        10000,
+        35000,
+        90500,
+    ]
+
+
```

Comment:
can you move this test to `tests/integ/test_functions.py`. We try not to add new tests in scala test suite since they were added when we first made function parity move between python and scala.